sdcm/sct_config.py

```diff
@@ -2018,8 +2018,8 @@ def verify_configuration(self):
 
         self._verify_data_volume_configuration(backend)
 
-        self._validate_seeds_number()
         if self.get('n_db_nodes'):
+            self._validate_seeds_number()
```

Comment:
I don't understand how this change is related and probably it should be in a different commit or explain in the commit message.

Reply:
linting test error:
```

[2023-08-21T09:58:05.833Z] ---- linting: ./test-cases/microbenchmarking/amazon_perf_simple_query_x86.yaml -----

[2023-08-21T09:58:05.833Z] Traceback (most recent call last):

[2023-08-21T09:58:05.833Z]   File "/tmp/jenkins/workspace/can_scylla-cluster-tests_PR-6512/./sct.py", line 766, in _run_yaml_test

[2023-08-21T09:58:05.833Z]     config.verify_configuration()

[2023-08-21T09:58:05.833Z]   File "/tmp/jenkins/workspace/can_scylla-cluster-tests_PR-6512/sdcm/sct_config.py", line 2021, in verify_configuration

[2023-08-21T09:58:05.833Z]     self._validate_seeds_number()

[2023-08-21T09:58:05.833Z]   File "/tmp/jenkins/workspace/can_scylla-cluster-tests_PR-6512/sdcm/sct_config.py", line 2092, in _validate_seeds_number

[2023-08-21T09:58:05.833Z]     num_of_db_nodes = sum([int(i) for i in str(self.get('n_db_nodes')).split(' ')])

[2023-08-21T09:58:05.833Z]   File "/tmp/jenkins/workspace/can_scylla-cluster-tests_PR-6512/sdcm/sct_config.py", line 2092, in <listcomp>

[2023-08-21T09:58:05.833Z]     num_of_db_nodes = sum([int(i) for i in str(self.get('n_db_nodes')).split(' ')])

[2023-08-21T09:58:05.833Z] ValueError: invalid literal for int() with base 10: 'None'
```


due to config file have no 'n_db_nodes' but function  self._validate_seeds_number() is still require it:
```
 def _validate_seeds_number(self):
        seeds_num = self.get('seeds_num')
        assert seeds_num > 0, "Seed number should be at least one"

        num_of_db_nodes = sum([int(i) for i in str(self.get('n_db_nodes')).split(' ')])
        assert not num_of_db_nodes or seeds_num <= num_of_db_nodes, \
            f"Seeds number ({seeds_num}) should be not more then nodes number ({num_of_db_nodes})"
```

Reply:
commit message has updated